### ISSUE TITLE
Ability to center an element type or highlight an element using title type.

### ISSUE THREAD

**Issue by DanUnderwood** [abhikmitra/ng-joyride#34]

**Comment by abhikmitra** [comment]
Can you give an example ?

**Comment by DanUnderwood** [comment]
I may wish to highlight a large area of the screen. In this case, a pop-over doesn't work as it either goes offscreen or is so far out of focus that the user doesn't see it. When trying to show large areas, it would be nice to have a title type be centered on the screen to draw the user's focus to that area.

**Comment by abhikmitra** [comment]
I get it . Joyrides are generally tied to specific elements. In your case your element covers a big area right ? So you wanna use titleType for it . correct ?

**Comment by DanUnderwood** [comment]
Yeah - but I worked around it. I was able to highlight a large area of the screen, but then items within that area became clickable, so I don't like that. When highlighting a larger area of the screen, the popover would also go off the screen. It was this that was causing me to think that a title type, or the ability to center the popover would be a nice option. Just something to possibly consider. Thanks.
